sr_health_check/package.xml

```diff
@@ -29,8 +29,9 @@
   <run_depend>rospy</run_depend>
   <run_depend>rqt_gui</run_depend>
   <run_depend>rqt_gui_py</run_depend>
-<export>
+
+<!-- <export>
```

Comment:
SHould we add a comment here explaining the reason why this is not being exported?

Reply:
we can - it is because of a bug in the code that needs to be fixed before release deployment. I can add this if you'd like

Reply:
I think it makes sense. Even adding a link to the relevant bug.

Reply:
added in 59b7446dbe5f3a82d221a834277edaf914cd869e